addons/userconfig/cba_settings.sqf

```diff
@@ -16,18 +16,31 @@ force ace_finger_enabled = true;
 
 force ace_gforces_enabledFor = 0;
 
-force ace_goggles_effects = 0;
-
 force ace_hearing_enableCombatDeafness = false;
 
 force ace_map_mapShowCursorCoordinates = true;
 
-force ace_medical_ai_enabledFor = 0;                      // Medical system not enabled for AI
```

Comment:
Add back key

Reply:
This should be set to 1, which is already default value, thus no need to set it here

Reply:
It's good to have for when someone decies to change it :)

And as reference is someone wants to look at our options